packages/vite/src/node/plugins/worker.ts

```diff
@@ -27,11 +28,18 @@ export async function bundleWorkerEntry(
   })
   let code: string
   try {
-    const { output } = await bundle.generate({
+    const {
+      output: [outputCode, ...outputChunks]
+    } = await bundle.generate({
       format,
       sourcemap: config.build.sourcemap
     })
-    code = output[0].code
+    code = outputCode.code
+    outputChunks.forEach((outputChunk) => {
+      if (outputChunk.type === 'asset') {
+        ctx.emitFile(outputChunk)
+      }
```

Comment:
outted is handle type chunk, use `registerAssetToChunk` register assets to chunks and generate assets metadata in manifest. 

But I think the worker's chunk should not go to the manifest plugin, so it should be OK not to 'registerassettochunk'

Reply:
maybe I need to test chunk lately.